monday/resources/columns.py

```diff
@@ -0,0 +1,13 @@
+from monday.query_joins import update_multiple_column_values_query, create_column
+
+from monday.resources.base import BaseResource
+
+
+class ColumnsResource(BaseResource):
+    def create_column(self, board_id):
+        query = create_column(board_id)
+        return self.client.execute(query)
+
+    def update_multiple_columns(self, item_id, board_id, column_values):
+        query = update_multiple_column_values_query(item_id, board_id, column_values)
```

Comment:
Additionally, the `update_multiple_column_values_query` method has an optional parameter `create_labels_if_missing` that is missing here - it needs to be added as an optional argument to this signature or calls to this will fail.

Reply:
Fixed.